### PR TITLE
Add Twitter link

### PR DESCRIPTION
**Pull request by rnicoll**

### CODE REVIEW

File: getting-started/index.html [hunk 1]
```diff
@@ -713,7 +713,8 @@ <h2><strong data-i18n="dgc-getting-started-paper-4-0">4. Print and fold your wal
 				<h4 data-i18n="dgc-community-modal-title" class="modal-title dogefont" id="community-modal-lable">COMMUNITY:</h4>
 				<h5 data-i18n="dgc-community-modal-get-involved">Get involved with Dogecoin! <br/><br/><span data-i18n="dgc-community-modal-participate">Participate in our Reddit community, chat on our IRC (Internet Chat) channel, or have fun on our forums.</span></h5>
 				<ul>
-					<li><a data-i18n="dogecoincom-reddit" href="http://www.reddit.com/r/dogecoin" target="_blank">Reddit Community</a><span data-i18n="dgc-community-modal-reddit-desc"> - The Main Hotspot of the Dogecoin Community.</span></li>
+					<li><a data-i18n="dogecoincom-reddit" href="https://www.reddit.com/r/dogecoin/" target="_blank">Reddit Community</a><span data-i18n="dgc-community-modal-reddit-desc"> - The Main Hotspot of the Dogecoin Community.</span></li>
+					<li><a data-i18n="dogecoincom-twitter" href="https://twitter.com/dogecoin" target="_blank">Twitter Community</a><span data-i18n="dgc-community-modal-twitter-desc"> - The Official Dogecoin Twitter Feed.</span></li>
```
Comment (patricklodder) [inline getting-started/index.html]: "Twitter" instead of "Twitter Community"

File: i18n/doge.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
     "dgc-fun-currency": "Much fun, such friendly internet currency.", 
     "dgc-set-apart": "Dogecoin has very amaze, much vibrant community! Made of friendly folks like me and you.",
     "dgc-reddit": "Reddit Community", 
+    "dgc-twitter": "Twitter Community",
```
Comment (patricklodder) [inline i18n/doge.json]: "Twitter"

File: i18n/en.json [hunk 3]
```diff
@@ -39,6 +39,7 @@
     "dgc-fun-currency": "The fun and friendly internet currency.", 
     "dgc-set-apart": "Dogecoin sets itself apart from other digital currencies with an amazing, vibrant community made up of friendly folks just like you.",
     "dgc-reddit": "Reddit Community", 
+    "dgc-twitter": "Twitter Community",
```
Comment (patricklodder) [inline i18n/en.json]: "Twitter"

**Review by patricklodder — COMMENTED** [review]
Not a community - it's a single twitter account.

File: getting-started/index.html (hunk 1 above)
Comment (rnicoll) [reply getting-started/index.html]:
```suggestion
					<li><a data-i18n="dogecoincom-twitter" href="https://twitter.com/dogecoin" target="_blank">Twitter</a><span data-i18n="dgc-community-modal-twitter-desc"> - The Official Dogecoin Twitter Feed.</span></li>
```

File: i18n/doge.json (hunk 2 above)
Comment (rnicoll) [reply i18n/doge.json]:
```suggestion
    "dgc-twitter": "Twitter",
```

File: i18n/en.json (hunk 3 above)
Comment (rnicoll) [reply i18n/en.json]:
```suggestion
    "dgc-twitter": "Twitter",
```

**Review by patricklodder — APPROVED** [review]

**Pull request merged**